tests/acceptance/features/bootstrap/GraphContext.php

```diff
@@ -1253,6 +1253,31 @@ public function theFollowingUsersShouldBeListedInFollowingGroups(TableNode $tabl
 		}
 	}
 
+	/**
+	 * @Then the user :user should be listed once in the group :group
+	 *
+	 * @param string $user
+	 * @param string $group
+	 *
+	 * @return void
+	 */
+	public function theUsersShouldBeListedOnceToAGroup(string $user, string $group): void {
+		$count = 0;
+		$members = $this->listGroupMembers($group);
+		$members = $this->featureContext->getJsonDecodedResponse($members);
+
+		foreach ($members as $member) {
+			if ($member['onPremisesSamAccountName'] === $user) {
+				$count++;
+			}
+		}
+		Assert::assertEquals(
+			1,
+			$count,
+			"The user '" . $user . "' has been added '" . $count . "' time in a group. "
```

Comment:
```suggestion
			"Expected count of user  " .  $user . "in a group " . $group . " to be 1 but found " . $count 
```
may be something like this ?